src/Tribe/Plugins_List.php

```diff
@@ -0,0 +1,126 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugins_List' ) ) {
+	/**
+	 * A list of Tribe's major plugins. Useful when encouraging users to download one of these.
+	 */
+	class Tribe__Plugins_List {
```

Comment:
That makes sense. I'd like to see Plugin_List stay separate. First because I could see us adding more info to it at some point that would make it more useful in other circumstances. Ideally we would only have one list of this sort of info to maintain. We already much of this info duplicated across our plugin suite in a few areas, so it would be awesome if we could start consolidating. Second, I could see a small possibility in the future where we make info like this info obtainable from the tec servers, and for that I am keen to keep that class separate with it's own get_list method.